transport/model/deployment.go

```diff
@@ -0,0 +1,43 @@
+package model
+
+import (
+	"time"
+
+	svcmodel "release-manager/service/model"
+
+	"github.com/google/uuid"
+)
+
+type CreateDeploymentInput struct {
+	ReleaseID     uuid.UUID `json:"release_id"`
+	EnvironmentID uuid.UUID `json:"environment_id"`
+}
+
+type Deployment struct {
+	ID               uuid.UUID `json:"id"`
+	ReleaseID        uuid.UUID `json:"release_id"`
+	ReleaseTitle     string    `json:"release_title"`
+	EnvironmentID    uuid.UUID `json:"environment_id"`
+	EnvironmentName  string    `json:"environment_name"`
+	DeployedByUserID uuid.UUID `json:"deployed_by_user_id"`
+	DeployedAt       time.Time `json:"deployed_at"`
+}
+
+func ToSvcCreateDeploymentInput(input CreateDeploymentInput) svcmodel.CreateDeploymentInput {
+	return svcmodel.CreateDeploymentInput{
+		ReleaseID:     input.ReleaseID,
+		EnvironmentID: input.EnvironmentID,
+	}
+}
+
+func ToDeployment(dpl svcmodel.Deployment) Deployment {
```

Comment:
Simplified structure is returned in transport layer. I think it is enough to return only release id and title and environment id and name, instead of the whole release and env struct.

```
{
    "id": "99c464e7-98d7-4ad0-9f3b-0bc5bfe0eb11",
    "release_id": "7f38395f-2e89-4c20-9e58-89447f212457",
    "release_title": "v1.0.0",
    "environment_id": "10374687-7bd4-45be-8703-a6e4a409ef0f",
    "environment_name": "production",
    "deployed_by_user_id": "aa24c1f3-df3b-40d9-b204-8a87fa065f5b",
    "deployed_at": "2024-05-26T10:15:40.788344+02:00"
}
```

Reply:
For now - OK; we can add more fields later.